srcpkgs/libupnp1.12/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'libupnp1.12'
+pkgname=libupnp1.12
+version=1.12.1
```

Comment:
1.14 exists... Does this really need 1.12? :(

Reply:
1.12 was the version listed in gerbera's readme. I'll try to build it against 1.14 and see if it works.

Reply:
It compiles and seems to work with 1.14, so I replaced the 1.12 package.